src/views/Streamer.vue

```diff
@@ -353,6 +371,25 @@ export default {
     setCodecs(codec) {
       this.codecs = codec;
     },
+    copyUrl() {
+      let copyText = document.getElementById('public-link');
+      let input = document.createElement('input');
+      input.setAttribute('value', copyText.innerText);
+      document.body.appendChild(input);
+      input.select();
+
+      let result = document.execCommand('copy');
+      const copyNotification = document.getElementById('copy-notification')
+      if (result) {
+        document.body.removeChild(input);
+        copyNotification.style.visibility = 'visible';
```

Comment:
Instead of changing visibility, make a variable in data and put a `v-if` on the HTML that is bound to that variable.

Reply:
okay! Fixed.